doc/conf.py

```diff
@@ -273,7 +273,7 @@ def _prevent_sphinx_circular_imports_bug():
     "matplotlib": ("https://matplotlib.org", None),
     # "dask": ("https://docs.dask.org/en/latest", None),
     # "numba": ("https://numba.pydata.org/numba-doc/latest", None),
-    "pint": ("https://pint.readthedocs.io/en/stable", None),
+    "pint": ("https://pint.readthedocs.io/en/latest", None),
```

Comment:
I had hoped this would include your fix in the `pint` docs but somehow it doesn't 😞  @marscher

Reply:
They merged it quite some time ago, but it hasn't been released yet, right? I'd guess latest is just link to the latest release, not unreleased main branch stuff :)

Reply:
I think `stable` is the current release and `latest` is actually the main branch (same as for weldx or asdf etc.)

The fix is in there but still doesn't work somehow..
https://pint.readthedocs.io/en/latest/developers_reference.html?highlight=to_compact#pint.Quantity.to_compact